src/sdk/model/sso/oidc-identity-provider-config-request.ts

```diff
@@ -0,0 +1,229 @@
+import { OIDCClientAuthType } from './__json__/oidc-client-auth-type';
+import { OIDCPromptType } from './__json__/oidc-prompt-type';
+
+/* istanbul ignore next: autogenerated */
+export class OIDCIdentityProviderConfigRequest {
+
+  private readonly _json: OIDCIdentityProviderConfigRequestJson;
+
+  constructor(OIDCIdentityProviderConfigRequest: OIDCIdentityProviderConfigRequest);
+  constructor(OIDCIdentityProviderConfigRequestJson: OIDCIdentityProviderConfigRequestJson);
+  constructor(firstParam: OIDCIdentityProviderConfigRequest | OIDCIdentityProviderConfigRequestJson) {
+    if (firstParam instanceof OIDCIdentityProviderConfigRequest) {
+      // Copy constructor
+      this._json = (firstParam as OIDCIdentityProviderConfigRequest).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as OIDCIdentityProviderConfigRequestJson;
+    }
+  }
+
+  /**
+   * Get enforce SSO.
+   * @returns {boolean}
+   */
+  get enforceSSO(): boolean {
+    return this._json.enforce_sso;
+  }
+
+  /**
+   * Get enabled.
+   * @returns {boolean}
+   */
+  get enabled(): boolean {
+    return this._json.enabled;
+  }
+
+  /**
+   * Get authorization url.
+   * @returns {string}
+   */
+  get authorizationUrl(): string {
+    return this._json.authorization_url;
+  }
+
+  /**
+   * Get logout URL.
+   * @returns {string}
+   */
+  get logoutUrl(): string | undefined {
+    return this._json.logout_url;
+  }
+
+  /**
+   * Get login hint.
+   * @returns {boolean}
+   */
+  get loginHint(): boolean | undefined {
+    return this._json.login_hint;
+  }
+
+  /**
+   * Get ui locales.
+   * @returns {boolean}
+   */
+  get uiLocales(): boolean | undefined {
+    return this._json.ui_locales;
+  }
+
+  /**
+   * Get token url.
+   * @returns {string}
+   */
+  get tokenUrl(): string {
+    return this._json.token_url;
+  }
+
+  /**
+   * Get user info url.
+   * @returns {string}
+   */
+  get userInfoUrl(): string | undefined {
+    return this._json.user_info_url;
+  }
+
+  /**
+   * Get backchannel supported.
+   * @returns {boolean}
+   */
+  get backchannelSupported(): boolean | undefined {
+    return this._json.backchannel_supported;
+  }
+
+  /**
+   * Get disable user info.
+   * @returns {boolean}
+   */
+  get disableUserInfo(): boolean | undefined {
+    return this._json.disable_user_info;
+  }
+
+  /**
+   * Get client id.
+   * @returns {string}
+   */
+  get clientId(): string {
+    return this._json.client_id;
+  }
+
+  /**
+   * Get client secret.
+   * @returns {string}
+   */
+  get clientSecret(): string {
+    return this._json.client_secret;
+  }
+
+  /**
+   * Get client auth.
+   * @returns {OIDCClientAuthType}
+   */
+  get clientAuth(): OIDCClientAuthType {
+    return this._json.client_auth;
+  }
+
+  /**
+   * Get issuer.
+   * @returns {string}
+   */
+  get issuer(): string | undefined {
+    return this._json.issuer;
+  }
+
+  /**
+   * Get default scope.
+   * @returns {string}

```

Comment:
| undefined reflected in return doc for the rest of the getters in this class that can have return type `undefined`

Reply:
Fixed